README.md

```diff
@@ -28,6 +28,20 @@ npm run serve
 
 You should now have the application running and accessible at <http://localhost:8080> (note: will run on a different port if 8080 is occupied, most common alternative port is 8081).
 
+## Building as web component
+To build the project as a web component, run the command `npm run build-component`. This will create a file in the `dist` folder named `license-chooser.min.js`. It can be used to load the web-component in any JS project. There is also a sample `demo.html` created.
+To be able to use the file it should either be rendered statically from the integrater's web-app or be published on a CDN. Following code can be used to integrate this in other apps:
+
+```
+<script src="https://unpkg.com/vue"></script>
+<!-- If the licenser-chooser file is hosted on http://127.0.0.1:8081 -->
+<script src="https://unpkg.com/@creativecommons/chooser/dist/license-chooser.js"></script>
+
+<license-chooser></license-chooser>
+```
+
+If the web component is intended to be built without header and footer, run `VUE_APP_CC_OUTPUT=embedded npm run build-component`
```

Comment:
By default, the web component should no longer include the header and footer. Is that the case?

Reply:
@brylie Yes, it seems so as the web component would be integrated into another app which would have its own header and footer.